scripts/fetch-lists.ts

```diff
@@ -0,0 +1,15 @@
+import fetch from 'node-fetch';
+
+export function fetchLists() {
+    return Promise.all([
+      'https://easylist.to/easylist/easylist.txt',
+      'https://easylist.to/easylist/easyprivacy.txt',
+      'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances.txt',
+      'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/badware.txt',
+      'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/filters.txt',
+      'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/privacy.txt',
+      'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/resource-abuse.txt',
+      'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/unbreak.txt',
+    ].map(url => fetch(url))).then(rs => Promise.all(rs.map(r => r.text())));
```

Comment:
Nice, there is also an alternative using the `async`/`await` syntax (you do not have to change to that, it's just for completeness' sake):
```js
return Promise.all([
]).map(async (url) => {
  const response = await fetch(url);
  const text = await response.text();
  return text;
});
```